src/components/WhaleSightingViewer/SightingList.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { getWhaleSightings, NewSighting } from "../../clients/apiClient";
+import "./SightingList.scss";
+
+export default function SightingList(props: {
+    pageNum: number,
+    sightingsList: NewSighting[],
+}) { 
```

Comment:
Can we create an interface for the props? Then when you use the values you don't need to call `props.pageNum`, it would just be `pageNum`
```suggestion
interface SightingListProps {
    pageNum: number;
    sightings: WhaleSighting[];
}

export default function SightingList({
    pageNum,
    sightings
}: SightingListProps) { 
```